test/Datadog.Trace.Tests/Logging/DatadogLoggingTests.cs

```diff
@@ -74,6 +77,131 @@ public void MessageTemplates_ReplacesArgumentsCorrectly()
             Assert.Equal("Warning level message with argument '123'", log.RenderMessage());
         }
 
+        [Fact]
+        public void RateLimiting_WhenLoggingIsOnSeparateLines_DoesntRateLimit()
+        {
+            const int secondsBetweenLogs = 60;
+            var rateLimiter = new LogRateLimiter(secondsBetweenLogs);
+            var logger = new DatadogSerilogLogger(_logger, rateLimiter);
+
+            var clock = new SimpleClock();
+            Clock.SetForCurrentThread(clock);
```

Comment:
You should make sure to dispose the return value at the end of the test
(same comment for other tests)

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1153/commits/05de3984613bd001f05f2c5f22b34db3ccb4d2e5 (including in `LogRateLimiterTests`)